modules/virt-configuring-ip-for-vm.adoc

```diff
@@ -0,0 +1,63 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/vm_networking/virt-attaching-vm-multiple-networks.adoc
+
+[id="virt-configuring-ip-for-vm_{context}"]
+= Configuring an IP address for a virtual machine
+
+You can use cloud-init to configure an IP address for a virtual machine. The IP address can be provisioned to be dynamic or static. 
+
+You can add or edit the cloud-init configuration of an existing virtual machine configuration. If you edit a running virtual machine, the virtual machine must be rebooted for the changes to take effect.
+
+.Prerequisites
+
+* DNS is configured via DHCP on the cluster.
+
+* A Linux bridge network attachment definition.
+
+* The virtual machine is connected to the bridge network. 
+
+// web console and CLI? Link to edit in web console?
+
+.Procedure
+
+. Edit or create a virtual machine configuration and include the cloud-init network details in the `spec.volumes.cloudInitNoCloud.networkData` field of the virtual machine configuration:
+
+.. To configure a dynamic IP, specify the interface name and the `dhcp` boolean:
++
+[source,yaml]
+----
+kind: VirtualMachine
+spec:
+...
+  volumes:
+  - cloudInitNoCloud:
+      networkData: |
+        version: 2
+        ethernets:
+          eth1: # or whatever interface the user wants to configure
```

Comment:
Don't say whatever!

Reply:
@phoracek - the 'attach vm to multiple networks' has a prereq of 
"A Linux bridge must be configured and attached on every node. See the node networking section for more information." 
Does that mean we should use bridge instead of ethernet in these examples, or am I missing something?

Reply:
The ethernet is fine. I just question whether we should repeat prerequisities for secondary networks here.

This is describing guest configuration. Guest configuration only requires a secondary network to be available, it does not care how is the secondary network configured (the how is described in the multiple networks guide).

By depending only on **a** secondary network available, we disentangle the dependency.